code-challenges.test.js

```diff
@@ -34,21 +56,93 @@ const object3 = { number: -7 }
 
 // b) Create the function that makes the test pass.
 
+// Psuedo code:
+// Start on the function by naming a function divideByThree
+// Write out all the syntax for the function
+// Use two if else statments
+// use the % modulator to check if a number is divisible by 3
+// return is divisible by 3 if true
+// return is not divisible by 3 if not true
+
+// const divideBYThree = () => {
+//     if( object.number % === 0){
+//         return `${object.number} is divisible by three.`
+//     } else {
+//         return`${object.number} is not divisibly by three`
+//     }
+// }
+
+// console.log(divideByThree(object1))
+
+// I couldnt get the test to pass. It says unexpected number when I console.logged it. I think something is wrong with my test i dont think its in my actual function. 
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// Psuedo code:
+// Start with describe a function
+// "It" will return all the words in the array capitalized
+// Make two expect statements one for the first const one for the second
+// run test using $yarn jest watch it fail
+
+// describe("capitilizer", () => {
+//     it(returns every string in the array in capital letters, () => {
+//         const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+//         const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+//         expect(capitilizer(randomNouns1)).toEqual(["STREETLAMP", "POTATO", "TEETH", "CONCLUSION", "NEPHEW"])
+//         expect(capitilizer(randomNouns2)).toEqual(["TEMPERATURE", "DATABASE", "CHOPSTICKS", "MANGO"])
+
+
+//     })
+// })
+
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
 // b) Create the function that makes the test pass.
 
+// Psuedocode: 
+// name a function "capitilizer"
+// put array as the argument we are inputting
+// use the .map property to iterate through each value
+// use the .toUpperCase to return the strings capitilized
+
+// const capitilizer = (array) => {
+//     return array.map(words()){
+//         return words.toUpperCase()
+//     })
+// }
+
+// console.log(capitalizer(randomNouns1))
+
+// I got as far as I could. Also could not get this one to work. My terminal says I am missin a paranthesis somewhere.
```

Comment:
This was a good attempt Wags! You are so close.  Pay close attention to your syntax. You have 1 too many parenthesis.